content/en/docs/_index.md

```diff
@@ -6,11 +6,11 @@ weight: 1
 
 ## About Falco
 
-Falco is a behavioral activity monitor designed to detect anomalous activity in your applications. Using powerful [system call capture](https://sysdig.com/blog/fascinating-world-linux-system-calls/) technology originally built by Sysdig. Falco lets you continuously monitor and detect container, application, host, and network activity, all in one place, from one source of data, with one set of [rules](rules).
+**Falco** is a runtime security tool designed for [OCI]-compliant containers. It uses powerful [system call capture][sys_call_capture] technology, originally built by [Sysdig], to monitor the [behavior](#behaviors) of applications—including container, application, host, and network activity—all in one place and from one source of data. You can specify which behavior you consider anomalous using Falco [rules].
 
-### What kind of behaviors can Falco detect?
+### Which behaviors can Falco detect? {#behaviors}
 
-Falco can detect and alert on any behavior that involves making [Linux system calls](http://man7.org/linux/man-pages/man2/syscalls.2.html). Falco alerts can be triggered by the use of specific system calls, their arguments, and by properties of the calling process. For example, you can easily detect when:
+Falco can detect and alert on any system behavior that involves [Linux system calls][sys_calls]. You can instruct Falco to trigger alerts not only the basis of which system calls are made but also of arguments to those calls and properties of the calling process. Here are some example anomalous behaviors that can trigger Falco alerts:
```

Comment:
You can instruct Falco to trigger alerts not only on the basis of which system calls are made but also of arguments to those calls and properties of the calling process.

Reply:
Please rewrite " of arguments to those calls and properties of the calling process". The statement is not clear. Do you mean "on arguments to those calls and the properties of the calling process"? Do you mean
You can instruct Falco to trigger alerts on the system calls as well as on the arguments to those calls and properties of the calling process.